src/Microsoft.SqlTools.ServiceLayer/TaskServices/TaskMetadata.cs

```diff
@@ -41,6 +41,16 @@ public class TaskMetadata
         /// </summary>
         public string DatabaseName { get; set; }
 
+        /// <summary>
+        /// Project path this task is created for
+        /// </summary>
+        public string ProjectPath { get; set; }
```

Comment:
can you share screenshots of how these new fields look in the Tasks panel? In the past for the DacFx operations, I would set the target location to display the dacpac/bacpac that was getting created, so that's an option to use here for the project that's getting updated

Reply:
This is what shows up in the task panel:

![Task](https://user-images.githubusercontent.com/32072322/131768017-ee2b5284-71ca-43c0-a9ac-de1cceb9427c.png)

Reply:
It would be great to look into improving this so that it's clear what project/dacpac/etc. was modified

Reply:
Can you make it so that the updated project shows so that it's clear what was updated, like Charles mentioned? For extract dacpac, the location of the dacpac is shown, which was added in this PR: https://github.com/microsoft/sqltoolsservice/pull/936
![image](https://user-images.githubusercontent.com/31145923/131885656-d7951be3-e165-4978-85d3-b0219f6640d7.png)